docs/general/session.md

```diff
@@ -35,4 +35,41 @@ backends can link the two sessions.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Session Events
+
+### Session Start Event
+
+![Experimental](https://img.shields.io/badge/-experimental-blue)
+
+`event.name`=`session.start`
+
+For instrumentation that tracks user behavior during user sessions, a `session.start` event MUST be emitted
+every time a session is created. When a new session is created as a continuation of a prior session,
+the `session.previous_id` SHOULD be included in the event. The values of `session.id` and `session.previous_id`
+MUST be different.
+
+When the `session.start` event contains both `session.id` and `session.previous_id` fields, the event then implies
+that the previous session has ended. If the session ID in `session.previous_id` has not yet ended via explicit
+`session.end` event, then the receiver SHOULD treat this continuation event as semantically equivalent to
+`session.end(session.previous_id)` and `session.start(session.id)`.
+
+| Body field                                                    | Type   | Description                                          | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability                                                        |
+|---------------------------------------------------------------|--------|------------------------------------------------------|-------------------------------------------------------------------------------------------------------|------------------------------------------------------------------|
+| [`session.id`](/docs/attributes-registry/session.md)          | string | The ID of the new session being started.             | `Required`                                                                                            | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
nit: as this is the "session.x" event do you really want / need the specific body fields to also be prefixed with "session."

Reply:
As part of the semantic conventions for events I am adding support for "alias" so that body fields (both in this case) could reference the global `session.id` and then provide an alias (name) for inclusion in the body along with overriding the inherited brief, note, examples etc. to further clarify what this specific usage is about. The general approach should be the narrowing of scope (from referenced attributes) rather than opening / broadening / loosening the global semantics.

Reply:
My preference is to keep the `session.` prefix for `session.id` here, as it helps easily connect with its usage in other places.  This event is sent only once in a while, so payload size should be less of a concern.

Reply:
I'm with @scheler on this one -- prefer to keep `session.id` here, at least until aliases are available. When that happens, I am happy to revisit.